sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -16,6 +16,15 @@
   - `UsernamePasswordCredential`
   - `VisualStudioCodeCredential`
 
+### Breaking Changes
+
+- Credential types supporting multi-tenant authentication will now throw `ClientAuthenticationError` if the requested tenant ID doesn't match the credential's tenant ID, and is not included in `additionally_allowed_tenants`. Applications must now explicitly add additional tenants to the `additionally_allowed_tenants` list, or add '*' to list, to enable acquiring tokens from tenants other than the originally specified tenant ID.
```

Comment:
Can you add a `BREAKING_CHANGES.md` file with the contents being added to other languages, and link to it from here?

Reply:
Added.